packages/flutter/lib/src/cupertino/dialog.dart

```diff
@@ -1234,6 +1235,11 @@ class CupertinoActionSheetAction extends StatefulWidget {
   /// Destructive buttons have red text.
   final bool isDestructiveAction;
 
+  /// The cursor that will be shown when hovering over the button.
+  ///
+  /// if null defaults to [SystemMouseCursors.click] on web else [MouseCursor.defer].
```

Comment:
```suggestion
  /// If null, defaults to [SystemMouseCursors.click] on web and [MouseCursor.defer] on other platforms.
```
https://github.com/flutter/flutter/blob/master/docs/contributing/Style-guide-for-Flutter-repo.md#use-correct-grammar